index.md

```diff
@@ -73,34 +81,32 @@ public class CartSummaryViewModel : IHandle<CartChangedMessage>
 }
 ```
 
-#### Works everywhere Xaml does
-Supports WPF, Silverlight, Windows Phone, Windows 8 including Universal apps.
-
-### and many more ...
-
-## Getting Started
-The quickest way to get started is down grab the latest package from [Nuget][nuget] and browse the [documentation][docs]. If you're running into trouble checkout our [support][support] section.
-
-
 ## Who's Behind It
 The core contributors to Caliburn.Micro are:
 
- - [Nigel Sampson][nigel] - Project coordinator and responsible for the ports to new Xaml platforms such as Windows 8 and Windows Phone 8.1 / Universal Apps.
+ - [Ken Tucker][kt] - New Maintainer, Microsoft MVP, User group leader, Mobile Developer
```

Comment:
I was a microsoft mvp from 2003 to 2016

Reply:
Thanks, I wasn't quite sure, I did a quick google search, but figured you would probably have your own feedback.

Reply:
@vb2ae Was there anything else you would like me to add/remove?

Reply:
that looks good